crates/bevy_core_pipeline/src/core_3d/main_opaque_pass_3d_node.rs

```diff
@@ -53,12 +54,24 @@ impl ViewNode for MainOpaquePass3dNode {
             depth_prepass,
             normal_prepass,
             motion_vector_prepass,
+            deferred_prepass,
             skybox_pipeline,
             skybox_bind_group,
             view_uniform_offset,
         ): QueryItem<Self::ViewQuery>,
         world: &World,
     ) -> Result<(), NodeRunError> {
+        let load = if deferred_prepass.is_none() {
+            match camera_3d.clear_color {
+                ClearColorConfig::Default => LoadOp::Clear(world.resource::<ClearColor>().0.into()),
+                ClearColorConfig::Custom(color) => LoadOp::Clear(color.into()),
+                ClearColorConfig::None => LoadOp::Load,
+            }
+        } else {
+            // If the deferred lighting pass has run, don't clear again in this pass.
```

Comment:
this seems a bit tenuous. if the deferred_prepass has run, we assume that the deferred lighting pass has run?

Reply:
We assume if the user put the `DeferredPrepass` component on the camera that some deferred lighting pass has run. 

Maybe there needs to be a more complex configuration so multiple user defined deferred lighting passes have a way of easily having only the first one clear. Especially since they could be defined in separate 3rd party plugins. And then clear in the forward pass if no deferred lighting pass ran.

Reply:
Maybe there’s some way of propagating it through the render attachments? But then the nodes would have to modify the render attachment structs and maybe that breaks node parallelism. Unless RwLock.

Reply:
I don’t think that should be done in this PR though.

Reply:
we do have a precedent for that with the postprocessing a/b textures. fine to do a separate pr though.